### DISCUSSION TITLE
Where to find Android arm32-bit armeabi-v7a apk ?

### DISCUSSION THREAD

**Discussion by Universalizer** [Alex313031/Thorium-Special#41] [Show and tell]
### Thanks for your greatest work,

Yesterday's, https://github.com/Alex313031/Thorium-Special/releases/tag/M110.0.5481.179

As mentioned titled here, "Android ARM64 & ARM32 Releases - M110.0.5481.179",

Is Android arm32-bit armeabi-v7a apk, available for major version 110 ?

Once again, Thank You.

**Comment by gz83** [comment]
> ### Thanks for your greatest work,
> Yesterday's, https://github.com/Alex313031/Thorium-Special/releases/tag/M110.0.5481.179
> 
> As mentioned titled here, "Android ARM64 & ARM32 Releases - M110.0.5481.179",
> 
> Is Android arm32-bit armeabi-v7a apk, available for major version 110 ?
> 
> Once again, Thank You.

@Universalizer 

arm32 version may be released in the near future

**Reply by Universalizer** [reply]
Thanks @gz83, for information.

**Comment by Alex313031** [comment]
@Universalizer Sorry, I am building it right now. I will add it to that release when it is ready.

**Comment by Alex313031** [comment]
@Universalizer It is up now. You can find it in the same release.

**Comment by Universalizer** [comment]
Thank You so much,

But, how to get rid of this "blinking red flash light on all border of screens" every few seconds ?

**Reply by gz83** [reply]
> Thank You so much,
> 
> But, how to get rid of this "blinking red flash light on all border of screens" every few seconds ?

@Universalizer 

This is caused by strict mode, please check whether strict mode is enabled on the phone

**Reply by Universalizer** [reply]
@gz83

> This is caused by strict mode, please check whether strict mode is enabled on the phone

### I had already checked, it is "already disabled on the phone".

Check and verify the screenshot image below,

![©Universalizer®Screenshot_20230316-HHMMSS_Settings Md](https://user-images.githubusercontent.com/116723165/225556812-93c67f8f-fe12-479f-8c6d-4bd124bc3995.png)

**This matter ("blinking red flash light on all border of screens" every few seconds) still exist, is their any another solution ? Please inform, thank you,
Still i will manage to continue with it.**

**Reply by gz83** [reply]
@Universalizer 

It may be that there is still a definition for strict mode in the code, we will investigate

**Reply by Universalizer** [reply]
@gz83

Thank you.

May i suggest one more matter or thing.

**Reply by gz83** [reply]
> @gz83
> 
> Thank you.
> 
> May i suggest one more matter or thing.

@Universalizer 

yes please speak out

**Reply by Universalizer** [reply]
@gz83,

Point of view only,
Optionally if possible,

Any plan for "focusing privacy",
i.e. Integration & merging of "uBo (uBlock Origin) open source APIs or SDKs", for blocking ads and trackers.

To the Developing Team and @Alex313031
Thanks to your efforts and dedicating work,
Well wishes for excellence.

**Reply by gz83** [reply]
> @gz83,
> 
> Point of view only, Optionally if possible,
> 
> Any plan for "focusing privacy", i.e. Integration & merging of "uBo (uBlock Origin) open source APIs or SDKs", for blocking ads and trackers.
> 
> To the Developing Team and @Alex313031 Thanks to your efforts and dedicating work, Well wishes for excellence.

@Universalizer 

Thank you for your suggestion, we will evaluate it based on various factors before deciding whether to add it to the Thorium browser

**Comment by Universalizer** [comment]
Optionally,
Any plan for merging & integrating "Bromite's bookmarks management features patch for offline backup & restore" ?

Remember, optional only, not mandatorily,

Ultimately, your choice and will.<hr>

By the way, good to see many Ungoogled Chromium patches with flag settings and some made by you from scratch. https://github.com/Alex313031/thorium/blob/7043a803e862c0b46f376a49cd2fbd305f3ee707/infra/PATCHES.md?plain=1
